salt/config/__init__.py

```diff
@@ -43,9 +43,10 @@
 
 try:
     import psutil
-    psutil.virtual_memory
+    if not hasattr(psutil, 'virtual_memory'):
+        raise ImportError('Version of psutil too old.')
```

Comment:
@cachedout You're welcome. I'd also learned this not long ago. :)

@guettli Not sure how this can be solved but a new issue is a better place to discuss that than this pull request. :)